src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx

```diff
@@ -44,9 +48,12 @@ const Wrapper = styled.div`
 const InnerWrapper = styled.div`
   margin: 0 auto;
   position: relative;
+  background: ${colorsV3.gray200};
+  padding: 0rem 1rem;
```

Comment:
this adds extra padding that we should not have. 
you can see the design here that left right we have 16px on both screens. 
https://www.figma.com/file/GQ1b2GBtO3SxGW2YaH7qng/Purchase-Flow-Checkout?node-id=327%3A1852

Reply:
oh, seems like the issue is on CheckoutPayment only